config/routes.rb

```diff
@@ -1,7 +1,5 @@
 # frozen_string_literal: true
 Rails.application.routes.draw do
-  # TODO(#45): add rack-cors middleware to streamline CORS config
-  # Adding CORS preflight routes here for now to unblock front-end dev
   match '/v0/*path', to: 'application#cors_preflight', via: [:options]
```

Comment:
I previously thought that the rack-cors gem would take care of handling preflight requests, but it turns out we still need to have a route/action to handle them. OPTIONS is the request method (instead of GET, POST, etc).

see https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS#Preflighted_requests for more info.